config_panel.toml

```diff
@@ -0,0 +1,203 @@
+        ask = "URL to the web client which / will redirect to."
+        type = "url"
+        help = "Basically Element instance the server should redirect to."
+        bind = "web_client_location:/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.client_base_url]
+        ask = "URL for client links within the email notifications."
+        type = "url"
+        help = "Used to be called 'riot_base_url', still supported"
+        bind = "client_base_url:/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.invite_client_location]
+        ask = "Web client location to direct users to during an invite."
+        type = "url"
+        help = "This is passed to the identity server as the org.matrix.web_client_location key. Defaults to unset, giving no guidance to the identity server."
+        bind = "invite_client_location:/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.enable_group_creation]
+        ask = "Allow non-server-admin Users to create Spaces?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'false'."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.experience.e2e_enabled_by_default]
+        ask = "End-to-End Encryption by default for locally-created Rooms"
+        type = "select"
+        choices = ["all", "invite", "off"]
+        help = '''
+        * "all": any locally-created room
+        * "invite": any room created with the "private_chat" or "trusted_private_chat" room creation presets
+        * "off": this option will take no effect
+        '''
+        bind = "encryption_enabled_by_default_for_room_type:/etc/matrix-__APP__/homeserver.yaml"
+
+    [client.welcome]
+    name = "Welcome User Experience"
+
+        [client.welcome.auto_join_rooms]
+        ask = "Auto Join new Users in following Rooms."
+        type = "string"
+        help = "Room should be public. If not existing, will be created as public and federated by default."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.welcome.autocreate_auto_join_rooms]
+        ask = "Create room for Auto Join if not existing."
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Will be created as public and federated by default. Can be customised with the settings auto_join_*."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.welcome.auto_join_rooms_for_guests]
+        ask = "Enable Auto Join Room for Guests?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'true'."
+        bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+        [client.welcome.password_enabled]
+        ask = "Enable Password Login?"
+        type = "boolean"
+        yes = true
+        no = false
+        help = "Defaults to 'true'."
+        bind = "password_config>enabled:/etc/matrix-__APP__/homeserver.yaml"
```

Comment:
Are you sure of this: `password_config>enabled` ?

Reply:
yep see https://github.com/Gredin67/synapse_ynh/blob/config-panel/conf/homeserver.yaml#L2167-L2170
I already tested this with synapse, and other bridge packages https://github.com/YunoHost-Apps/mautrix_whatsapp_ynh/blob/expose-config-panel/config_panel.toml